src/import-data/ImportDataOverview.ts

```diff
@@ -69,41 +71,53 @@ const ResponseFragment = (props: ResponseFragmentProps): ReactNode => {
   ]);
 };
 
-interface ImportDataOverviewProps {
-  header: string;
-  isDataset: boolean;
-  isProtectedData: boolean;
+const getTitleForImportRequest = (importRequest: ImportRequest): string => {
```

Comment:
The OOP programmer in me wishes this was defined inside or alongside `ImportRequest` instead of here. If it was, then it wouldn't need to be moved whenever its usage moves. But I haven't seen that done in terra-ui and don't know if that's a common practice in JS/TS in general.

Reply:
I thought of it more as presentation vs data model, so it didn't occur to me to move it into the import-types module alongside `ImportRequest`.

As for defining it _inside_ `ImportRequest`, the limitation there is that `ImportRequest` is currently only an interface, not a class. So the actual `ImportRequest` objects are plain JS objects without any custom methods. We _could_ turn them into classes/sub-classes. I don't think would be particularly unusual for JS in general. But it would be a bit of a break with convention in Terra UI code, given Terra UI's history of using functional programming patterns.